### ISSUE TITLE
Row order being sent incorrectly

### ISSUE THREAD

**Issue by bliswebagency** [solspace/craft3-freeform#1355]
### What happened?

Re-arranging the order of fields enough times causes them to lose their ability to save their order.

Please see attached screenshot for the payload sent during saving. It looks like it might be treating the "row.order" value as a string and appending 1 instead of incrementing. When it reaches the int limit the DB sets it to the highest value and it becomes stuck and unable to sort.

![Screenshot 2024-06-19 at 2 15 14 PM](https://github.com/solspace/craft-freeform/assets/158676/2f36d1cb-df97-46c6-aa0a-c08bab12466f)

### Errors and Stack Trace (if available)

```shell
Please see attached output and screenshots

--- sample rows from database
*************************** 23. row ***************************
         id: 715
     formId: 1
   layoutId: 30
      order: 0
dateCreated: 2024-06-19 03:02:33
dateUpdated: 2024-06-19 03:57:36
        uid: d75ed07b-c4de-4bc2-91bc-6897a1bc1b1c
*************************** 24. row ***************************
         id: 716
     formId: 1
   layoutId: 30
      order: 11111111
dateCreated: 2024-06-19 03:02:33
dateUpdated: 2024-06-19 03:57:36
        uid: 9d6ebe33-db4e-4508-9b1b-06712e475ce9
*************************** 25. row ***************************
         id: 717
     formId: 1
   layoutId: 29
      order: 11111111
dateCreated: 2024-06-19 03:02:33
dateUpdated: 2024-06-19 03:57:36
        uid: 162a27cc-f673-4ea9-be7a-f47191890e57
*************************** 26. row ***************************
         id: 718
     formId: 1
   layoutId: 7
      order: 11111111
dateCreated: 2024-06-19 03:02:33
dateUpdated: 2024-06-19 03:57:36
        uid: 2443519f-cddb-4a0b-ab87-e8426515ecdf
*************************** 27. row ***************************
         id: 719
     formId: 1
   layoutId: 30
      order: 2111111111
dateCreated: 2024-06-19 03:02:33
dateUpdated: 2024-06-19 03:57:36
        uid: 1fef8ed9-9ceb-4e63-8819-4ce8772fc4cb
*************************** 28. row ***************************
         id: 720
     formId: 1
   layoutId: 30
      order: 1011111111
dateCreated: 2024-06-19 03:02:33
dateUpdated: 2024-06-19 03:57:36
        uid: 214a0c9a-77c9-442e-9b32-d5e03e4d8a7a
*************************** 29. row ***************************
         id: 721
     formId: 1
   layoutId: 25
      order: 2147483647
dateCreated: 2024-06-19 03:02:33
dateUpdated: 2024-06-19 03:57:36
        uid: 1ba61c2e-92b8-4dad-8704-aedc086b3b0f
*************************** 30. row ***************************
         id: 722
     formId: 1
   layoutId: 28
      order: 2147483647
dateCreated: 2024-06-19 03:02:33
dateUpdated: 2024-06-19 03:57:36
        uid: 9b6a72d9-e6ac-42a2-b227-57c5b98c0767
*************************** 31. row ***************************
         id: 723
     formId: 1
   layoutId: 26
      order: 2147483647
dateCreated: 2024-06-19 03:02:33
dateUpdated: 2024-06-19 03:57:36
        uid: 914c4f65-3f95-4d8c-b93c-fcbdd3912883
*************************** 32. row ***************************
         id: 724
     formId: 1
   layoutId: 29
      order: 2147483647
dateCreated: 2024-06-19 03:02:33
dateUpdated: 2024-06-19 03:57:36
        uid: 2acfcac0-53cc-4847-8d6f-5d8652175587
*************************** 33. row ***************************

----
```

### How can we reproduce this?

1. Move fields around to change their order then save
2. Repeat moving and saving enough times
3. Eventually the field order will not be retained
4. Check db to find value of freeform_forms_rows for fields have hit the int limit (2147483647)

PHP: 8.2.20
MariaDB: 10.6.18

### Freeform Edition

Pro

### Freeform Version

5.4.0

### Craft Version

4.10.2 Pro

### When did this issue start?

Unsure

**Comment by kjmartens** [comment]
Hi @bliswebagency,

We'll check into this and see if we can duplicate. Roughly how many times did it take you to hit this? 🙂

**Comment by kjmartens** [comment]
This should now be fixed in Freeform 5.5.0. Please check and see if that's the case for you. 🙂

**Comment by bliswebagency** [comment]
Thanks for the fix. We can confirm the ordering is working again after the update. 

But just noting that we had to shuffle around each field for existing forms for them to 'reset' their order values and after then they would re-order correctly again. Just passing it on in case others encounter this same scenario.
